### DISCUSSION TITLE
Historico de Skins

### DISCUSSION THREAD

**Discussion by ghost** [leogianfagna/IssueTracker#998] [Ideia]
### Servidor afetado

Henesys

### Qual o propósito da ideia?

Isso facilita tanto a busca entre os players quanto a espera pelo retorno das caixas.

### Descrição da ideia

Expandir a boutique com todas as skins já lançadas, ou adicioná-las na wiki, ajuda os jogadores a visualizar as skins, facilitando sua procura.

**Comment by leogianfagna** [comment]
Criar um menu com todas as coleções e adicionar para ver cada uma com: `/crazycrates preview deuses nickname`
